src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsLowTrust.cs

```diff
@@ -1,182 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@brianrob, these tests weren't included in projects, and when I tried including them, they failed to compile / run.  Do we need to keep them?

Reply:
This looks all desktop specific which is probably why they weren't included.  They can be removed.

Reply:
Ok, thanks.